activesupport/test/time_travel_test.rb

```diff
@@ -162,4 +162,13 @@ def test_time_helper_travel_with_time_subclass
       assert_equal DateTime.now.to_s, DateTimeSubclass.now.to_s
     end
   end
+
+  def test_time_helper_freeze_time
```

Comment:
This is not a deterministic test. This can still pass without the correct implementation, you can get same formatted time 3 times.